src/components/Button/Button.tsx

```diff
@@ -1,86 +1,56 @@
-import React from 'react'
-import './Button.scss'
-
-export interface ButtonProps {
-	/**
-	 * Classes to be applied to the button
-	 */
-	className?: string
-
-	/**
-	 * Is button disabled?
-	 * @Default false
-	 */
-	disabled?: boolean
+import React, { useMemo } from 'react'
 
+export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
 	/**
 	 * Text of the button
 	 */
 	text?: string
 
-	/**
-	 * Either a button text or a react component.
-	 */
-	children?: React.ReactNode
-
-	/**
-	 * Style to be applied to the button.
-	 */
-	style?: React.CSSProperties
-
 	/**
 	 * Button varient.
 	 * @Default 'primary'
 	 */
-	variant?: 'primary' | 'secondary' | 'tertiary' | 'alert' | 'pause' | 'save' | 'white-filled' | 'facebook'
+	variant?: 'primary' | 'secondary' | 'negative' | 'outline'
 
 	/**
-	 * Function to be called when hit a button
+	 * Icon in Button
 	 */
-	onClick: (e?: any) => void
+	suffixIcon?: JSX.Element
+}
 
-	/**
-	 * Type of the button.
-	 */
-	type?: 'button' | 'submit' | 'reset'
+const Button: React.FC<ButtonProps> = (props) => {
+	const classesName = useMemo(() => {
+		switch (props.variant) {
+			case 'primary':
+				return 'bg-blue-500 border-transparent hover:bg-blue-300 hover:border-red-100 disabled:bg-gray-500 '
 
-	/**
-	 * Id of the div item.
-	 */
-	id?: string
+			case 'secondary':
+				return 'bg-red-400 border-transparent hover:bg-red-500 hover:border-red-100 disabled:bg-gray-500 '
 
-	/**
-	 * To check which button is clicked with same onClick function.
-	 */
-	dataId?: string
+			case 'negative':
+				return 'bg-gray-600 border-transparent hover:bg-gray-50/30 hover:border-gray-50/30 disabled:bg-gray-50 '
 
-	/**
-	 * ID used to identify the button during testing.
-	 */
-	dataTestId?: string
-}
+			case 'outline':
+				return 'text-gray-50 bg-transparent border-gray-400 hover:border-gray-200 disabled:text-gray-200 '
+
+			default:
+				return ''
+		}
+	}, [props.variant])
 
-const Button: React.FC<ButtonProps> = React.forwardRef((props, ref: any) => {
 	return (
 		<button
+			className={
+				`align-center flex flex-row ${
+					props.suffixIcon ? 'justify-between' : 'justify-center'
```

Comment:
@frannkenstein justify-between can be the default style as it will have no effect on only one child div